MobiFlight/Config/MultiplexerDriver.cs

```diff
@@ -31,12 +31,8 @@ public bool isInitialized()
 
         public bool registerClient()
         {
-            bool res = false;
-            if (_initCounter > 0) {
-                _initCounter++;
-                res = true;
-            }
-            return res;
+            _initCounter++;
+            return true;
         }
 
         public bool unregisterClient()
```

Comment:
This should be a `private` message because this is "inner" state that no one should worry about outside.